helpers/prism.js

```diff
@@ -0,0 +1,430 @@
+import { css, unsafeCSS } from 'lit-element/lit-element.js';
+
+window.Prism = window.Prism || {};
+Prism.manual = true;
+
+const prismLocation = 'https://s.brightspace.com/lib/prismjs/1.28.0';
+//const prismLocation = '/node_modules/prismjs'; // for local debugging
+
+// If adding a language, check its Prism dependencies and modify languageDependencies below if necessary
+export const codeLanguages = new Map();
+codeLanguages.set('arduino', 'Arduino');
+codeLanguages.set('armasm', 'ARM Assembly');
+codeLanguages.set('bash', 'Bash');
+codeLanguages.set('c', 'C');
+codeLanguages.set('clike', 'C-like');
+codeLanguages.set('cpp', 'C++');
+codeLanguages.set('csharp', 'C#');
+codeLanguages.set('css', 'CSS');
+codeLanguages.set('haskell', 'Haskell');
+codeLanguages.set('java', 'Java');
+codeLanguages.set('javascript', 'JavaScript');
+codeLanguages.set('json', 'JSON');
+codeLanguages.set('kotlin', 'Kotlin');
+codeLanguages.set('latex', 'LaTeX');
+codeLanguages.set('markup', 'Markup');
+codeLanguages.set('matlab', 'MATLAB');
+codeLanguages.set('plain', 'Plain Text');
+codeLanguages.set('python', 'Python');
+codeLanguages.set('r', 'R');
+codeLanguages.set('racket', 'Racket');
+codeLanguages.set('sql', 'SQL');
+codeLanguages.set('wolfram', 'Wolfram');
```

Comment:
I was wondering about this when I first wrote it, but admittedly I wasn't sure so I avoided the array initialization approach because I was averse to creating arrays that would be temporary.  So... I decided to do up the 3 methods of initializing and run a test - [benchmark can be run here](https://jsbench.me/wrl4ohuosv/1).

This is the result: 
Initialize with `set`: _fastest_
Initialize with `array`: 39.21% slower
Initialize with `object`: 63.87% slower

However, I'm not sure how much trust to put it the tool, and the result may vary depending on the size of the initial dataset. I think it's fast regardless, so putting the perf aside, I am happy to switch it if we prefer, otherwise I'll just leave it as-is.

Reply:
Interesting. I'm more than a bit surprised that the object/array initialization methods are slower. You would think the engine would treat them equivalently. Good to know I suppose.

Reply:
I suppose that in the slower cases, we are effectively creating an intermediate data structure, and the Map constructor then processes that and populates it's own the same way that `set` would. 🤷‍♂️

Reply:
Yeah, that would make sense. I suppose the constructor exists more for converting existing iterables into maps than for actual initialization.

Reply:
Crazy! Yeah I'm good with whatever. I personally find the object initialization the easiest to read and it'll end up making our code smaller without N `codeLanguages.set(` calls, but 🤷 .